Developer-Guide.md

```diff
@@ -277,10 +286,131 @@ To perform analysis on Kata logs, use the
 [`kata-log-parser`](https://github.com/kata-containers/tests/tree/master/cmd/log-parser)
 tool.
 
+See also [Setting up a debug console](#setting-up-a-debug-console).
+
 # Appendices
 
 ## Checking Docker default runtime
 
 ```
 $ sudo docker info 2>/dev/null | grep -i "default runtime" | cut -d: -f2- | grep -q runc  && echo "SUCCESS" || echo "ERROR: Incorrect default Docker runtime"
 ```
+
+## Setting up a debug console
+
+By default, it is not possible to login to a virtual machine since this could
+be sensitive from a security perspective. However it also requires additional
+packages in the rootfs which means the image used to boot each virtual machine
+would be larger.
+
+If you wish to be able to login to a virtual machine that hosts your
```

Comment:
Lines 306-307 suggested rewrite: If you want to login to a virtual machine that hosts your containers, complete the following steps, which assume a rootfs image.